xml/http_servers.xml

```diff
@@ -1689,13 +1689,21 @@
     <param pos="0" name="service.component.product" value="JBossWeb"/>
     <param pos="2" name="service.component.version"/>
   </fingerprint>
-  <fingerprint pattern="^Servlet/[\d\.]+; JBossAS-(.*)$">
+  <fingerprint pattern="^Servlet\/[\d\.]+; JBossAS-(.*)$">
     <description>JBoss AS</description>
-    <example>Servlet/3.0; JBossAS-6</example>
+    <example service.version="6">Servlet/3.0; JBossAS-6</example>
     <param pos="0" name="service.vendor" value="Red Hat"/>
     <param pos="0" name="service.product" value="JBoss"/>
     <param pos="1" name="service.version"/>
   </fingerprint>
+    <fingerprint pattern="^JBoss-EAP\/(\d+)$">
+    <description>JBoss EAP</description>
+    <example service.version="7">JBoss-EAP/7</example>
+    <param pos="0" name="service.vendor" value="Red Hat"/>
+    <param pos="0" name="service.family" value="JBoss"/>
+    <param pos="0" name="service.product" value="JBoss EAP"/>
```

Comment:
Should all (or perhaps just "more") of the references to JBoss in recog be changed to match this?  Similarly with the family.

Otherwise this looks ready to go.